CMP1903_A1_2324/Game.cs

```diff
@@ -15,8 +16,42 @@ internal class Game
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
 
+        public int sumtotal = 0;
         //Methods
-
+        public int[] roll3()
+        {
+            //create a new die 
+            Die Die1 = new Die();
+            //roll the die
+            Die1.Roll();
+            //wait 1 thousandth of a second to allow device clock to change to allow for multiple random numbers
+            Thread.Sleep(1);
+            //repeat for second die
+            Die Die2 = new Die();
+            Die2.Roll();
+            Thread.Sleep(1);
+            //repeat for third die minus the delay as it is unnecissary 
+            Die Die3 = new Die();
+            Die3.Roll();
+            //save the rolled values as variables for easier usage
+            int dice1 = Die1.rolledvalue;
+            int dice2 = Die2.rolledvalue;
+            int dice3 = Die3.rolledvalue;
+            //display the rolled numbers to the user
+            Console.WriteLine("rolls");
+            Console.WriteLine(dice1);
+            Console.WriteLine(dice2);
+            Console.WriteLine(dice3);
```

Comment:
You can use a for loop to get rid of the duplicate code and only use one dice object